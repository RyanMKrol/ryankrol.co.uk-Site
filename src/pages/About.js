import React from 'react';

import logo from './../assets/logo.svg';

import './About.css';

function About() {
  return (
    <div className="page-body">
      <div className="About">
        <img src={logo} className="About-logo" alt="logo" />
        <p>
          About Page - Under Development!
        </p>
      </div>
    </div>
  );
}

export default About;
