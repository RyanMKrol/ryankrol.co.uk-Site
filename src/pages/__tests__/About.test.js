import React from 'react'
import { shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'

import About from './../About'

// Render Tests

it('renders About only when rendered', () => {
    const about = shallow(<About />)
    expect(toJson(about)).toMatchSnapshot()
})

it('does not crash when rendered', () => {
    expect(() => mount(<About />)).not.toThrow()
})
