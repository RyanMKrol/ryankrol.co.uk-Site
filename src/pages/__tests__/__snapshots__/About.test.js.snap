// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders About only when rendered 1`] = `
<div
  className="container"
>
  <div
    className="About"
  >
    <img
      alt="logo"
      className="About-logo"
      src="logo.svg"
    />
    <p>
      About Page - Under Development!
    </p>
  </div>
</div>
`;
